### PR TITLE
Fix integration tests not running on Github Actions

### LINKED ISSUE #129: after moving to github actions, integration tests are no longer run
Need to figure out how to run dynalite in github actions. or ideally run against both dynalite and dynamodb local.

### PR DESCRIPTION
**Pull request by hanscg**
Closes #129 

Integration tests now run with both dynalite and dynamodb-local.
Unfortunately service can't be set with matrix so the code has to be duplicated

### CODE REVIEW

File: .github/workflows/workflow.yml [hunk 1]
```diff
@@ -19,12 +19,48 @@ jobs:
...
+    needs: lock
+    strategy:
+      matrix:
+        python-version: ["3.7", "3.8", "3.9", "3.10"]
+    runs-on: "ubuntu-latest"
+    services:
+      dynamodb:
+        image: amazon/dynamodb-local:latest
+        ports:
+          - 8000:8000
```
Comment (dimaqq) [inline .github/workflows/workflow.yml:59]: Ugh that's a bummer that services don't grok matrix....
I guess there's another alternative:
* 1 job
* has 2 services on 2 ports
* matrix which port is used by `pytest`

Perhaps that's too clever though -- hard to tell what fails if it's `port 8001` 🤔

**Review by dimaqq — APPROVED** [review]

File: .github/workflows/workflow.yml [hunk 2]
```diff
@@ -19,12 +19,48 @@ jobs:
...
-  pytest:
+  pytest-dynalite:
     needs: lock
     strategy:
       matrix:
         python-version: ["3.7", "3.8", "3.9", "3.10"]
     runs-on: "ubuntu-latest"
+    services:
+      dynalite:
+        image: dimaqq/dynalite:latest
```
Comment (dimaqq) [inline .github/workflows/workflow.yml:30]: Hey folks, I've moved the image to ghcr.io though it's documented poorly 😅 
https://github.com/dimaqq/dynalite-docker

**Review by dimaqq — APPROVED** [review]
Awesome, Hans!

**Review by ojii — APPROVED** [review]
thank you

**Pull request merged**